src/test/java/racingcar/MyApplicationTest.java

```diff
@@ -0,0 +1,67 @@
+package racingcar;
+
+import static camp.nextstep.edu.missionutils.test.Assertions.assertSimpleTest;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import camp.nextstep.edu.missionutils.test.NsTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class MyApplicationTest extends NsTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = {"\n", " ", "  "})
+    @DisplayName("입력 레벨에서 잘못된 자동차이름 입력을 테스트합니다.")
+    void 예외_테스트_잘못된_자동차이름_입력(String input) {
+        assertSimpleTest(() ->
+                assertThatThrownBy(() -> runException(input, "1"))
+                        .isInstanceOf(IllegalArgumentException.class)
+        );
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"\n", " ", "  "})
+    @DisplayName("입력 레벨에서 잘못된 시행횟수 입력을 테스트합니다.")
+    void 예외_테스트_잘못된_시행횟수_입력(String input) {
+        assertSimpleTest(() ->
+                assertThatThrownBy(() -> runException("java", input))
+                        .isInstanceOf(IllegalArgumentException.class)
+        );
+    }
```

Comment:
```suggestion
  @ParameterizedTest
  @ValueSource(strings = {"\n", " ", " "})
  @DisplayName("입력 레벨에서 잘못된 시행횟수 입력을 테스트합니다.")
  void 예외_테스트_잘못된_시행횟수_입력(String input) {
      assertSimpleTest(() -> 
          assertThatThrownBy(() -> runException("java", input))
              .isInstanceOf(IllegalArgumentException.class)
              .hasMessageContaining(ErrorMessages.INVALID_RANGE_TRY_COUNT.getMessage())
      );
  }
```

처럼 hasMessageContaining() 메서드를 추가하여 발생하는 예외의 메시지가 기대하는 메시지와 일치하는지도 추가로 검증하는 것은 어떻게 생각하시나요? 이를 통해 단순히 예외 타입뿐만 아니라 올바른 상황에서 적절한 메시지가 출력되는지까지 확인할 수 있을 것 같아요!

Reply:
좋은 제안입니다. 저도 메세지까지 검증할 수 있는지는 알지 못했어요. 다음주에 시도해볼게요